Cargo.toml

```diff
@@ -27,6 +27,7 @@ bytes          = { version = "1.0", optional = true, default-features = false }
 chrono         = { version = "0.4.6", optional = true, default-features = false }
 futures-util   = { version = "0.3", optional = true }
 heapless       = { version = "0.8", optional = true }
+moka           = { version = "0.12.3", features = ["future"] }
```

Comment:
Shouldn't this new `moka` dependency be optional and included bythe `unstable-client-transport` feature?